doc/whats_new/v1.0.rst

```diff
@@ -167,6 +167,13 @@ Changelog
   :class:`~sklearn.semi_supervised.LabelPropagation`.
   :pr:`19271` by :user:`Zhaowei Wang <ThuWangzw>`.
 
+
+:mod:`sklearn._iterative.py`
+.................................
+- |Fix| Fix ValueError triggered when setting the estimator as PLSRegression() in 
+  :class: `sklearn._iterative.py: func: _impute_one_feature`
```

Comment:
Maybe a higher-level entry might be clearer to users?
```suggestion
- |Fix| :class:`cross_decomposition.PLSRegression` can now be used as an estimator for 
  :class:`impute.IterativeImputer`.
```

Reply:
And we will need a non-regression test that should ensure that it is actually working in this PR while failing in the branch `main`

Reply:
I got back to the original issue, and it looks like some issues (like [`SkuaD01/scikit-learn#1`](https://github.com/SkuaD01/scikit-learn/pull/1/files)) implemented a test for this.

@isaack-mungui and @SkuaD01: you probably could work together to have your changes integrated in `mean`. 🙂

Reply:
@jjerphan Just to gain clarity, should I integrate changes in `mean` or follow through with @glemaitre's requested changes first?

Reply:
You can proceed with @glemaitre's request (I misspelled `main` and wrote `mean`).

Reply:
Got it. Thanks!